### DISCUSSION TITLE
Is there a way to hide names below the icons?

### DISCUSSION THREAD

**Discussion by pr3dd1** [Ross-Patterson/Portals-Desktop-Organization#78] [Q&A] [answered]
as said ...

**Comment by Ross-Patterson** [comment]
At this time no, however this should be fairly easy to add as a setting,

**Comment by Squaredude** [comment, marked as answer]
I was wondering the same thing. Not sure if it's something you added after this question, but I figured out a way to do it:

On Customize => Styling => Portal Content => Item Text Height = 0
